web/app/templates/card.html

```diff
@@ -0,0 +1,58 @@
+{% macro body() -%}
+	{% block card %}
+	{% if json %}
+
+		{% if json['media'] and json['media']['data'] and json['media']['data'][0]['iframe'] %}
+			<div class="card has-media">
+		{% elif json['images'] and json['images']['data'] %}
+			<div class="card has-images">
+		{% else %}
+			<div class="card no-media">
+		{% endif %}
+			<div class="header">
+				{% if json['favicon'] %}
+					<img class="favicon" src="{{ json['favicon'] }}"/>
+				{% endif %}
+
+				{% if json['provider_url'] %}
+				<div class="website-name">
+					<p >{{ json['provider_url'] }}</p>
+				</div>
+				{% endif %}
+			</div>
+			<div class="left-col">
+				{% if json['media'] and json['media']['data'] and json['media']['data'][0]['iframe'] %}
+					<div class="media"> {{ json['media']['data'][0]['iframe']|safe }}</div>
+				{% elif json['images'] and json['images']['data'] %}
+					<div class="images">
+	<!-- 				{% for i in range(json['images']['data']|length) %}
```

Comment:
incase we want some slider image thing in the future where you can pick the images :p